src/entity/Company.ts

```diff
@@ -25,4 +30,11 @@ export class Company {
 
   @Column({ type: 'text', array: true, default: [] })
   domains: string[];
+
+  @Column({
+    type: 'enum',
+    enum: CompanyType,
+    default: CompanyType.Business,
+  })
```

Comment:
Type should be string on psql side, we only use enum to validate in code and gql. This allows us to adjust the type more freely without changing column type.

Reply:
See other examples eg. `user_transacton.status`